tests/copy_directory/empty_directory.bzl

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Creates an empty directory."""
+
+def _empty_directory_impl(ctx):
+    out = ctx.actions.declare_directory(ctx.attr.name)
+    ctx.actions.run_shell(
+        outputs = [out],
+        command = "mkdir -p $@",
```

Comment:
"-p" might not work on windows (https://stackoverflow.com/questions/905226/what-is-equivalent-to-linux-mkdir-p-in-windows)

However, the change already makes it work in more cases on linux (when name = "a/b").

I wonder if such PRs should be our top priority.

Reply:
PS this might work with just `ctx.actions.do_nothing`. I know Bazel creates some directory for the action, not sure if output directory is among them.

Reply:
It's a test-only rule, so I think it's reasonable to require bash on windows for it, as many of our tests already do.

re top priority - it's a step towards making it possible to update Google's internal skylib to 1.3.0.

re `ctx.actions.do_nothing` - unfortunately, we require an action that generates `out`.